src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -211,6 +218,7 @@ DatePicker.propTypes = {
   inputRef: PropTypes.object,
   onClose: PropTypes.func,
   datetime: PropTypes.bool,
+  onKeyDown: PropTypes.func.isRequired,
```

Comment:
why you mark this as required ? maybe there will be not needed to pass from parent this function. It shouldnt be required.

Reply:
done